lf-queue/README.md

```diff
@@ -1,8 +1,4 @@
-# Multithreaded Bounded Lock-free Queue
+# lfq
+This program has been modified using C11 Atomics to support other processor architectures and has rewritten the code to solve the data race issues, enabling it to pass the runtime detection of Thread Sanitizer.
```

Comment:
Instead of mentioning "This program has been modified using C11 Atomics," you shall give a general overview for the rewritten program. i.e., some wording as following:
> The lock-free queue implemented is based on the paper [A Scalable, Portable, and Memory-Efficient
Lock-Free FIFO Queue](https://drops.dagstuhl.de/opus/volltexte/2019/11335/pdf/LIPIcs-DISC-2019-28.pdf) by Ruslan Nikolaev.
> Nikolaev's paper provides an overview of various methods for implementing lock-free queues. This implementation specifically adopts the SCQ (Scalable Circular Queue) approach, which is designed for bounded queues. However, this approach can be easily extended to support unbounded FIFO queues capable of storing an arbitrary number of elements.

You shall check Nikolaev's paper as well.